### ISSUE TITLE
the converge problem

### ISSUE THREAD

**Issue by azhangwei** [AlfredXiangWu/face_verification_experiment#102]
i try to use the mfm layer, but i find the net can't converge ,  i want to know the influenc of the mfm fuction for the net's convergence.

**Comment by AlfredXiangWu** [comment]
You can decrease the learning rate for training. Besides, you can also use the pre-trained model to fine-tune light CNN on your own datasets.
